src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -571,5 +571,11 @@ internal override bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedW
 
             return false;
         }
+
+        internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions, DiagnosticBag diagnostics)
+        {
+            Type.CheckAllConstraints(conversions, SyntaxNode.Location, diagnostics);
```

Comment:
>CheckAllConstraints [](start = 17, length = 19)

This call is a substitute for the check that would be performed in `Binder.ConstructNamedType` if not suppressed. That check uses `conversions.WithNullability(true)` if the nullable feature is enabled.

Reply:
OK, sounds like I should check out Binder.ConstructNamedType or maybe other implementations of AfterAddingTypeMembersChecks and try to imitate where it calls conversions.WithNullability(true).